app/controllers/aid_applications/base_controller.rb

```diff
@@ -10,5 +10,29 @@ def current_aid_application
       @_current_aid_application ||= current_organization.aid_applications.find(params[:aid_application_id])
     end
     helper_method :current_aid_application
+
+    def ensure_eligible
```

Comment:
these ensure_<status> methods weren't necessary for these stories, right? using the aid_applications#show to redirect to the right place was sufficient i think? it seems like reasonable behavior to restrict what tab a user can continue to based on the status, but i was just curious if there was a problem happening that prompted you to add this now?

Reply:
They weren't necessary for the stories. But given that I was pushing people to more places, I got a little paranoid that someone would be bumped along to the wrong place. So I did a sorta belt (redirect) -and -suspenders (access control) to ensure that we don't bump someone to the wrong place now or in the future.